#!/usr/bin/env python
PACKAGE = "performance_tests"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cpp_publish_rate",    int_t,    0, "Publish Frequency of cpp publisher", 10,  100, 100000)
gen.add("python_publish_rate",    int_t,    0, "Publish Frequency of python publisher", 10,  100, 100000)


exit(gen.generate(PACKAGE, "performance_tests", "PublishRates"))
